quarkchain/cluster/grpc_client.py

```diff
@@ -0,0 +1,40 @@
+import logging
+import argparse
+import grpc
+
+from quarkchain.generated import grpc_pb2
+from quarkchain.generated import grpc_pb2_grpc
+
+HOST = "localhost"
+PORT = "50051"
+
+
+class GrpcClient:
+    def __init__(self, channel):
```

Comment:
I think the constructor should take the host and port as arguments and build the channel inside